MonMaraicher/Features/FarmerDetail/FarmerDetailsViewModelTests.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  FarmerDetailsTests.swift
+//  MonMaraicherTests
+//
+//  Created by Yves Charpentier on 11/03/2024.
+//
+
+import XCTest
+@testable import MonMaraicher
+
+final class FarmerDetailsViewModelTests: XCTestCase {
```

Comment:
Ce code devrait pas etre ici car deja merge sur develop avec la PR precedente. Donc vu que tu as merge il faut que tu pull main puis tu te mets sur cette branch et tu rebase sur main. Puis tu push

Reply:
Autre astuce pour eviter ca dans tous les cas pour les lecteurs, il faut que cette PR soit pas en comparaison avec la branch main mais avec la branch de la PR precedente. Bien sur, une fois merge tu repasses cette branch en comparaison avec main ;)

Reply:
I'm going to try without breaking everything